modules/op-release-notes-1-11.adoc

```diff
@@ -235,3 +235,41 @@ With this update, {pipelines-title} General Availability (GA) 1.11.1 is availabl
+
+[id="release-notes-1-11-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.2
+
+With this update, {pipelines-title} General Availability (GA) 1.11.2 is available on {OCP} 4.12 and later versions.
+
+This update includes an updated version of the `tkn` command line tool. You can download the updated version of this tool at the following locations:
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-arm64.tar.gz[Linux on ARM (aarch64, arm64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-windows-amd64.zip[Windows]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-macos-amd64.tar.gz[macOS]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-macos-arm64.tar.gz[macOS on ARM]
+
+If you installed the `tkn` command line tool using RPM on {op-system-base-full}, use the `yum update` command to install the updated version.
+
+[id="fixed-issues-1-11-2_{context}"]
+=== Fixed issues
+
+* Before this update, the `tkn pac resolve -f` command did not detect the existing secret for authentication with the Git repository. With this update, this command successfully detects the secret.
+
+* With this update, you can use `--v1beta1` flag in the `tkn pac resolve` command. Use this flag if the pipeline run is generated with the `v1beta1` API version schema.
+
+* Before this update, the `tkn pr logs` command failed to display the logs for a pipeline run if this pipeline run was created using a resolver. With this update, the command displays the logs.
+
+* With this update, the SHA digest of the `git-init` image corresponds to version 1.12.1, which is the current released version of the image
+
+== Known issues
+
+* If you use the Bundles resolver to define a pipeline run and then use the `tkn pac resolve --v1beta1` command for this pipeline run, the command generates incorrect YAML output. The `kind` parameter for the bundle is set to `Task` in the YAML output. As a workaround, you can set the correct value in the YAML data manually. Alternatively, you can use the version of the `tkn` tool included with {pipelines-shortname} version 1.12.0 or later.
```

Comment:
```suggestion
* If you use the Bundles resolver to define a pipeline run and then use the `tkn pac resolve --v1beta1` command for this pipeline run, the command generates incorrect YAML output. The `kind` parameter for the bundle is set to `Task` in the YAML output. As a workaround, you can set the correct value in the YAML data manually. Alternatively, you can use the version of the `tkn` tool included with {pipelines-shortname} version 1.12.0 or later OR opc pac resolve --v1beta1 will work
```

I mean i just want to say that 
another workaround is
`opc pac resolve --v1beta1`